source/layouts/layout.erb

```diff
@@ -7,11 +7,17 @@
 	<%= partial "/layouts/partials/footer" %>
 
 	<!--   Core JS Files   -->
-	<script src="/assets/js/core/jquery.3.2.1.min.js" type="text/javascript"></script>
-	<script src="/assets/js/core/tether.min.js" type="text/javascript"></script>
-	<script src="/assets/js/core/bootstrap.min.js" type="text/javascript"></script>
+	<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```

Comment:
I'm not sure our webserver will be okay with cross origin requests.  We may need to verify this ourselves.

Reply:
With regards to the CDN stuff, I don't think there will be a huge performance gain at all really, Since the site is really lightweight and static. 
For the sake of simplicity I'll revert to self hosted files.